plugins/woocommerce-blocks/packages/components/checkbox-control/index.tsx

```diff
@@ -32,6 +32,8 @@ export const CheckboxControl = ( {
 	hasError = false,
 	checked = false,
 	disabled = false,
+	errorId,
+	errorMessage,
```

Comment:
was showing an error in console.

Reply:
I think it would be more correct to ensure we don't pass invalid props to this component thats why the warning happens.

Reply:
Yeah I will strip them out in Form.